domain/src/main/java/com/teamagam/gimelgimel/domain/messages/SendGeoMessageInteractor.java

```diff
@@ -7,59 +7,40 @@
 import com.teamagam.gimelgimel.domain.map.entities.mapEntities.GeoEntity;
 import com.teamagam.gimelgimel.domain.map.entities.mapEntities.PointEntity;
 import com.teamagam.gimelgimel.domain.map.entities.symbols.PointSymbol;
-import com.teamagam.gimelgimel.domain.map.repository.DisplayedEntitiesRepository;
-import com.teamagam.gimelgimel.domain.map.repository.GeoEntitiesRepository;
 import com.teamagam.gimelgimel.domain.messages.entity.MessageGeo;
 import com.teamagam.gimelgimel.domain.messages.repository.MessagesRepository;
 import com.teamagam.gimelgimel.domain.notifications.repository.MessageNotifications;
 import com.teamagam.gimelgimel.domain.user.repository.UserPreferencesRepository;
 
-import rx.Observable;
-
 @AutoFactory
-public class SendGeoMessageInteractor extends SendBaseGeoMessageInteractor<MessageGeo> {
+public class SendGeoMessageInteractor extends SendMessageInteractor<MessageGeo> {
 
-    private final DisplayedEntitiesRepository mGeoDisplayedRepo;
     private final String mMessageText;
     private final PointGeometry mMessageGeometry;
     private final String mMessageType;
-    private GeoEntity mGeoEntity;
 
     SendGeoMessageInteractor(
             @Provided ThreadExecutor threadExecutor,
             @Provided UserPreferencesRepository userPreferences,
             @Provided MessagesRepository messagesRepository,
             @Provided MessageNotifications messageNotifications,
-            @Provided GeoEntitiesRepository geoEntitiesRepository,
-            @Provided DisplayedEntitiesRepository geoDisplayedRepo,
             String text,
             PointGeometry pg,
             String type) {
-        super(threadExecutor, userPreferences, messagesRepository, messageNotifications,
-                geoEntitiesRepository);
-        mGeoDisplayedRepo = geoDisplayedRepo;
+        super(threadExecutor, userPreferences, messagesRepository, messageNotifications);
         mMessageText = text;
         mMessageGeometry = pg;
         mMessageType = type;
     }
 
     @Override
-    protected Observable<MessageGeo> buildUseCaseObservable() {
-        return storeGeoEntityObservable()
-                .doOnNext(geoEntity -> mGeoEntity = geoEntity)
-                .doOnNext(mGeoDisplayedRepo::show)
-                .flatMap(e -> super.buildUseCaseObservable());
-    }
-
-    @Override
     protected MessageGeo createMessage(String senderId) {
-        return new MessageGeo(null, senderId, null, mGeoEntity);
+        return new MessageGeo(null, senderId, null, createGeoEntity());
     }
 
-    @Override
-    protected GeoEntity createGeoEntity(String id) {
+    private GeoEntity createGeoEntity() {
         PointSymbol symbol = createSymbolFromType(mMessageType);
-        return new PointEntity(id, mMessageText, mMessageGeometry, symbol);
+        return new PointEntity("not_used", mMessageText, mMessageGeometry, symbol);
```

Comment:
Add a constant "not_used" to Constants? It appears in SendImageMessageInteractor too...